app/interactors/file_attachments/create_interactor.rb

```diff
@@ -24,7 +24,10 @@ def call
 private
 
   def check_for_issues
-    issues = Requirements::FileAttachmentUploadChecker.new(params[:file], params[:title]).issues
+    issues = Requirements::FileAttachmentUploadChecker.new(params[:file]).issues
+    title_issues = Requirements::FileAttachmentTitleChecker.new(params[:title]).issues
```

Comment:
I wonder if we can have this be just one checker and that checker calls the title one?

Thinking a little bit a head I think we basically have two almost identical checkers for file and title but one requires file to exist other doesn't. Could be that we're able to have two simple classes FileAttachmentCreateChecker and FileAttachmentEditChecker and they both call out to file and title checker classes

Reply:
I like @benthorner's suggestion to create 'upload_issues' and 'update_issues' methods.  It will probably mean the class name should be more generic - FileAttachmentChecker I guess?

Reply:
Sounds good 👍

Reply:
This is now done - let me know if it's ok.

Reply:
That looks good - only thing that might be a bit confusing is 'update_issues' sounds like we're going to update them 😆. We use the 'pre_' prefix in other places, which might help with the ambiguity here i.e. 'pre_update_issues' and 'pre_create_issues'. Sorry, should have thought of this earlier :-|.

Reply:
Fair point - now updated :)